### PR TITLE
Update basic usage docs

### LINKED ISSUE #151: Update basic usage example docs
replace `.create` and `.transfer` with (`.prepare`, `.fulfill`, `.send`)

### PR DESCRIPTION
**Pull request by sbellem**
closes #151

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -110,53 +110,112 @@ who claims to be the signee.
...
-                                          signing_key=alice.signing_key,
-                                          asset=bicycle)
+   prepared_creation_tx = bdb.transactions.prepare(
+        operation='CREATE',
+        owners_before=alice.verifying_key,
+        asset=bicycle,
+        metadata=metadata,
+   )
 
 The ``creation_tx`` dictionary should be similar to:
```
Comment (ttmc) [inline docs/usage.rst]: `creation_tx` changed its name to `prepared_creation_tx`

**Comment by ttmc** [comment]
I noticed that some of the Python command lines begin with `>>>` and others begin with nothing. I prefer the nothing version because it's easier to copy & paste, but regardless of which version you prefer, you should stick with one or the other.

**Comment by ttmc** [comment]
I have no more comments to make about this pull request (at this time).

**Comment by sbellem** [comment]
> I noticed that some of the Python command lines begin with >>> and others begin with nothing. You should stick with one or the other.

Indeed!

Was not so sure on to handle this.

1. Making the code easy to copy and paste: YES!
2. Making it clear what is the command, versus what is the output: YES!

1 and 2 conflict :confused:

**Pull request merged**